public/app/plugins/datasource/prometheus/result_transformer.ts

```diff
@@ -103,6 +95,15 @@ export function transformV2(
     return { ...dataFrame, meta: { ...dataFrame.meta, dataTopic: DataTopic.Annotations } };
   });
 
+  const [heatmapResults, framesWithoutTableHeatmapsAndExemplars] = partition<DataFrame>(
```

Comment:
does this also need to happen for browser mode queries in `transform`?

Reply:
Nope, the heatmap is parsed after the exemplars are in that function so we are good to go.